cmd/lncli/commands.go

```diff
@@ -525,7 +532,31 @@ func sendPaymentCommand(ctx *cli.Context) error {
 var ShowRoutingTableCommand = cli.Command{
 	Name:        "showroutingtable",
 	Description: "shows routing table for a node",
-	Action:      showRoutingTable,
+	Usage:       "showroutingtable [--table]",
+	Flags: []cli.Flag{
+		cli.BoolFlag{
+			Name:  "table",
+			Usage: "Show the routing table in table format. Print only a few first symbols of id",
+		},
+		cli.BoolFlag{
+			Name:  "human",
+			Usage: "Simplify output to human readable form. Output lightning_id partially. Only work with --table option.",
```

Comment:
At a glance, based on the description of `human` compared to that of `table`, it doesn't seem that there's a meaningful difference between the two. 

If this is incorrect, then I think the description should be updated to precisely state the difference.